metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -374,7 +374,9 @@ def get_connection_object_page(
         if "errors" in query_data:
             errors = query_data["errors"]
             if all(
-                error.get("extensions", {}).get("severity", None) == "WARNING"
+                # The format of the error messages is highly unpredictable, so we have to
```

Comment:
Nice! Our culture on connectors needs to be one of _zero trust_ for objects we receive from foreign APIs. How can we move toward doing this ~~~everywhere~~~?